app/views/organisations/index.html.erb

```diff
@@ -0,0 +1,34 @@
+<% content_for :title, "Departments, agencies and public bodies - GOV.UK" %>
+
+<h1>Departments, agencies and public bodies - GOV.UK</h1>
```

Comment:
We might want to put hardcoded strings like this into a locale file so that, in the future, we can add translations into other languages.